target/linux/ath79/dts/ar9331_glinet_gl-mifi.dts

```diff
@@ -0,0 +1,152 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9331.dtsi"
+
+/ {
+	compatible = "glinet,gl-mifi", "qca,ar9331";
+	model = "GL.iNet GL-MIFI";
```

Comment:
Based on GL.iNet's page I'd adjust the capitalization to GL-MiFi ...
https://www.gl-inet.com/products/gl-mifi/

Reply:
Should be addressed in 753db9a